src/UnigramWordPredictor.java

```diff
@@ -100,8 +108,9 @@ public void train(Scanner scanner) {
    */
   public String predictNextWord(List<String> context) {
     // TODO: Return a predicted word given the words preceding it
-    // Hint: only the last word in context should be looked at
-    return null;
+    Random random = new Random();
+
+    return context.get(random.nextInt(context.size()));
```

Comment:
This doesn't quite work: we want to be choosing from our neighborMap, not from the context.